.github/workflows/ci.yaml

```diff
@@ -155,8 +155,6 @@ jobs:
                       bzlmodEnabled: true
                     - folder: e2e/npm_link_package
                       bzlmodEnabled: true
-                    - folder: e2e/npm_translate_lock
-                      bzlmodEnabled: true
```

Comment:
This e2e should now run twice on ci for Bazel 6, with and without bzlmod enabled.